### DISCUSSION TITLE
How to store image layers from docker build on the host machine?

### DISCUSSION THREAD

**Discussion by obergner** [abiosoft/colima#714] [Q&A] [answered]
We have a fairly complex project that uses Docker containers for its build process. Some of the Docker images that we use to improve the developer experience are fairly time-consuming to build. 

It _seems_ as if Colima caches the image layers generated by `docker build ...` in `/var/lib/docker`, **inside the VM**, and that the cached image layers **do not survive restarts of the Mac host**. Is this correct?

If so, how can I configure Colima to cache image layers on the host machine instead?

**Comment by abiosoft** [comment, marked as answer]
It does survive the restart of the macOS host. Are you having a contrary experience?

**Reply by obergner** [reply]
@abiosoft You are right: it **does** seem to survive restarts. I just did several tests again. It seems that one of the base images we use in the build has a high update frequency, we led to frequent rebuilds. Nothing to do with reboots. Thanks
